docs/certbot.rst

```diff
@@ -0,0 +1,52 @@
+Certbot options
+```````````````
+
+The certbot playbook
+~~~~~~~~~~~~~~~~~~~~
+
+As a convenience, the Plone Ansible Playbook kit includes a separate
+playbook that will install certbot-nginx and create certificates as necessary for specified hostnames.
+
+The certbot playbook currently only supports Debian-family target servers.
+
+To use the certbot playbook, edit your ``local-configure.yml`` file to add a ``certbot_hosts`` list variable containing an entry for each hostname for which you wish to get a certbot certificate:
+
+.. code-block:: yaml
+
+   certbot_hosts:
+     - one.mcsmith.org
+     - two.mcsmith.org
+
+Run the playbook as you would the main playbook, adding whatever command-line switches you need (like ``-k`` or ``-K``):
+
+.. code-block:: console
+
+    ansible-playbook -k certbot.yml
+
+This will first install ``python3-certbot-nginx`` from the certbot/certbot ppa.
```

Comment:
What's "ppa"? Is there a link to it?

Reply:
Personal Package Archive; standard debian mechanism for maintaining an additional package source. In this case, it's certbot's.

Reply:
```suggestion
This will first install ``python3-certbot-nginx`` from the certbot/certbot Personal Package Archive.
```